back/app/serializers/web_api/v1/notifications/event_upcoming_serializer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class WebApi::V1::Notifications::EventUpcomingSerializer < WebApi::V1::Notifications::NotificationSerializer
+  attribute :event_id do |object|
+    object.event&.id
+  end
+
+  attribute :event_title_multiloc do |object|
+    object.event&.title_multiloc
+  end
+
+  attribute :event_start_at do |object|
+    object.event&.start_at
+  end
+
+  attribute :project_title_multiloc do |object|
+    object.project&.title_multiloc
+  end
```

Comment:
I think we can remove the safe navigation operator here. If any of those attributes is missing, something is off, and we should fail hard.

```suggestion
  attribute :event_id do |object|
    object.event_id
  end

  attribute :event_title_multiloc do |object|
    object.event.title_multiloc
  end

  attribute :event_start_at do |object|
    object.event.start_at
  end

  attribute :project_title_multiloc do |object|
    object.project.title_multiloc
  end
```